packages/node/src/integrations/linkederrors.ts

```diff
@@ -42,26 +43,29 @@ export class LinkedErrors implements Integration {
    * @inheritDoc
    */
   public setupOnce(): void {
-    addGlobalEventProcessor((event: Event, hint?: EventHint) => {
-      const self = getCurrentHub().getIntegration(LinkedErrors);
+    addGlobalEventProcessor(async (event: Event, hint?: EventHint) => {
+      const hub = getCurrentHub();
+      const self = hub.getIntegration(LinkedErrors);
+      const stackParser = stackParserFromOptions(hub.getClient<NodeClient>()?.getOptions());
+
       if (self) {
-        const handler = self._handler && self._handler.bind(self);
-        return typeof handler === 'function' ? handler(event, hint) : event;
```

Comment:
Looks like this was added [here](https://github.com/getsentry/sentry-javascript/pull/2742) but seems no longer relevant because node has it's own `LinkedErrors` integration.